### PR TITLE
Fix get subscriptions and split into two endpoints

### PR DESCRIPTION
**Pull request by kilimandzsaro**

### CODE REVIEW

File: openapi_server/controllers/subscription_controller.py [hunk 1]
```diff
@@ -7,25 +7,56 @@
...
+    if get_children:
+      for child in get_children.get("children"):
+        children.append(child.get("externalId"))
+
+    subscriptions = []
+    for child in children:
+      get_subscription_param = {
+            "where": {
+              "child": {
+                "externalId": str(child)
```
Comment (Septimus4) [inline openapi_server/controllers/subscription_controller.py:42]: I don't think the str is needed around ids

File: openapi_server/controllers/subscription_controller.py [hunk 2]
```diff
@@ -7,25 +7,56 @@
...
-    return subscription
+      subscriptions.append(g.indykite_graph_client.execute(get_subscription_query, get_subscription_param))
+    return subscriptions
+
+
+def user_child_subscriptions_get(child_id):
+  get_subscription_param = {
+    "where": {
+      "child": {
+        "externalId": str(child_id)
```
Comment (Septimus4) [inline openapi_server/controllers/subscription_controller.py:54]: same

File: openapi_server/controllers/subscription_controller.py (hunk 1 above)
Comment (kilimandzsaro) [reply openapi_server/controllers/subscription_controller.py]: removed

File: openapi_server/controllers/subscription_controller.py (hunk 2 above)
Comment (kilimandzsaro) [reply openapi_server/controllers/subscription_controller.py]: removed

**Review by Septimus4 — APPROVED** [review]

**Pull request merged**